user/serializer.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+
+from user.models import UserProfile  
+from rest_auth.serializers import UserDetailsSerializer
+
+class UserSerializer(serializers.ModelSerializer):
+
+    #phone = serializer.PhoneNumberField(source="user.phone")
+    #address = serializers.CharField(source="user.phone")
+
+    class Meta(UserDetailsSerializer.Meta):
+        fields = UserDetailsSerializer.Meta.fields + ('phone','address')
+
+    def update(self, instance, validated_data):
+        profile_data = validated_data.pop('user', {})
+        phone = profile_data.get('phone')
+        address = profile_data.get('address')
+
+        instance = super(UserSerializer, self).update(instance, validated_data)
```

Comment:
better to do `super().update(instance, validated_data)`, no need to pass arguments to `super`